eralchemy/sqla.py

```diff
@@ -13,9 +13,20 @@
 
 def relation_to_intermediary(fk):
     """Transform an SQLAlchemy ForeignKey object to it's intermediary representation. """
+
+    # In the cases where schema name is provided, _column_tokens[1] is
```

Comment:
Can we extract all of these to a another function to make clear that the comment is about left_column name?

Reply:
Done.